modules/operator-unmanaged-hpa.adoc

```diff
@@ -6,21 +6,105 @@ Horizontal Pod Autoscalers (HPAs) are now included with the `Clair`, `Quay`, and
 
 As HPA is configured by default to be managed, the number of `Clair`, `Quay`, and `Mirror` pods is set to two. This facilitates the avoidance of downtime when updating or reconfiguring {productname} through the Operator or during rescheduling events.
 
+[NOTE]
+====
+There is a known issue when disabling the `HorizontalPodAutoscaler` component and attempting to edit the HPA resource itself and increase the value of the `minReplicas` field. When attempting this setup, `Quay` application pods are scaled out by the unmanaged HPA and, after 60 seconds, the replica count is reconciled by the {productname} Operator. As a result, HPA pods are continuously created and then removed by the Operator. 
```

Comment:
"There is a known issue xxxx" ->
Question: 
can we list the issue number here? I mean something like:  There is a known issue [PROJQUAY-6474](https://issues.redhat.com/browse/PROJQUAY-6474)
how do we usually do

Reply:
Yes. Done.